### ISSUE TITLE
Regenerated ISO with only spoiler log change and didnt crash, ended up soft locking

### ISSUE THREAD

**Issue by TotinoBoy** [LagoLunatic/wwrando#402]
So I've been doing a wind waker run and wanted to check if the overlook island cave was a required item or if I could just ignore it but I had generted no spoiler log originally, so I went and regenerted it with the spoiler log turned on to check it however after generation it still let me play the same save with my same items so I assumed nothing was wrong.

After a few hours of playing I fully ran out of locations to look for items and going back to the randomizer apparently regenerating just the spoiler log change causes items to change location? If thats the cause why did it let me continue playing? Why no like checksum or hash to prevent it from opening? This save is softlocked now and moot :(

**Comment by wooferzfg** [comment]
If you generate with a spoiler log, you get a different seed even with the same seed name. This is to prevent cheating in races.

**Comment by LagoLunatic** [comment]
The no spoiler log option changing item locations is intended behavior and is mentioned when you hover over the option:

> Prevents the randomizer from generating a text file listing out the locations of all items in this seed. (Also changes where items are placed in this seed.)
> Generating a spoiler log is highly recommended even if you don't intend to use it, just in case you get completely stuck.

This is done so that people can't cheat in races as easily as clicking a single checkbox.

> If thats the cause why did it let me continue playing? Why no like checksum or hash to prevent it from opening? This save is softlocked now and moot :(

So like storing a hash of the generated ISO's seed plus item-changing settings somewhere in the save file, and refusing to load the save if it doesn't match the ISO you're currently playing? That's an interesting idea. Would have to find some free space in the save file.

**Comment by TotinoBoy** [comment]
> The no spoiler log option changing item locations is intended behavior and is mentioned when you hover over the option:
> 
> > Prevents the randomizer from generating a text file listing out the locations of all items in this seed. (Also changes where items are placed in this seed.)
> > Generating a spoiler log is highly recommended even if you don't intend to use it, just in case you get completely stuck.
> 
> This is done so that people can't cheat in races as easily as clicking a single checkbox.
> 
> > If thats the cause why did it let me continue playing? Why no like checksum or hash to prevent it from opening? This save is softlocked now and moot :(
> 
> So like storing a hash of the generated ISO's seed plus item-changing settings somewhere in the save file, and refusing to load the save if it doesn't match the ISO you're currently playing? That's an interesting idea. Would have to find some free space in the save file.

yes something to prevent iso's from loading a different seeded save file.
